guides/howtos/Multi tenancy with foreign keys.md

```diff
@@ -27,7 +27,7 @@ defmodule MyApp.Repo do
   @impl true
   def prepare_query(_operation, query, opts) do
     cond do
-      opts[:skip_org_id] || opts[:schema_migration] ->
+      opts[:skip_org_id] || opts[:ecto_query] ->
```

Comment:
```suggestion
      opts[:skip_org_id] || opts[:ecto_query] in [:schema_migrations, :preload] ->
```

Reply:
So if we add more types in the future, it doesn't mean they get automatically skipped.

Reply:
It should be in [:schema_migration] (singular)